tst.sh

```diff
@@ -1,11 +1,23 @@
 #!/bin/sh
-while getopts "f:" arg; do
+mag=$'\e[1;35m'
+cyn=$'\e[1;36m'
+end=$'\e[0m'
+
+while getopts "f:k:" arg; do
   case $arg in
     f) FILE=$OPTARG;;
+    k) FORK=$OPTARG;;
   esac
 done
 
-npx ganache-cli -e 1000 -q & 2> /dev/null
+if [ -n "$FORK" ] 
+then
+  printf "%s\n\n" "${mag} Running fork:${end} ${cyn}$FORK${end}"
```

Comment:
coloring doesn't work for me
I get this print on my console:

$\e[1;35m Running fork:$\e[0m $\e[1;36mistanbul$\e[0m

Reply:
Weird... ok. I tried something else. Could u try again? If it doesn't work, I'm gonna remove it =/

Reply:
![image](https://user-images.githubusercontent.com/12406924/68732873-f309c000-060f-11ea-8141-18b46930c23a.png)